addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -151,23 +150,28 @@
                 <t t-set="is_member" t-value="slide.channel_id.is_member"/>
                 <t t-set="can_access" t-value="aside_slide.is_preview or is_member or slide.channel_id.can_publish"/>
                 <li class="p-0 pb-1">
-                    <a t-att-href="'/slides/slide/%s' % (slug(aside_slide)) if can_access else '#'"
-                        t-att-class="'o_wslides_lesson_aside_list_link d-flex align-items-center px-2 py-1 text-decoration-none %s%s' % (('bg-100 active' if aside_slide == slide else ''), 'text-muted' if not can_access else '')">
-                        <i t-if="is_member" t-att-id="'o_wslides_lesson_aside_slide_check_%s' % (aside_slide.id)"
-                            t-att-class="'mr-2 fa text-left %s' % ('text-success fa-check-circle' if channel_progress[aside_slide.id].get('completed') else 'text-600 fa-circle')">
-                        </i>
-                        <t t-call="website_slides.slide_icon">
+                    <div t-att-class="'o_wslides_lesson_aside_list_link d-flex align-items-center p-1 %s%s' % (('bg-100 active' if aside_slide == slide else ''), 'text-muted' if not can_access else '')"
+                        t-att-data-id="slide.id"
+                        t-att-data-completed="1 if slide_completed else 0">
```

Comment:
That's a bit strange, doesn't it support True/False?

Reply:
Isn't there some magic that does not add the attribute when the value is False, meaning it's equivalent to undefined in the data, meaning it works anyway?

(I tried greping "else 0" in xml files and there are not many occurrences, would like to find a more standard way to do this).

Reply:
see discord, dumb std, removed